db/migrate/20230719092447_create_posts.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts, &:timestamps
+  end
```

Comment:
- [ ] It's fine implementing the posts table, but kindly ensure that all columns as stated in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) are added appropriately. ℹ️